sidecar/config/config.go

```diff
@@ -38,9 +38,10 @@ type Service struct {
 
 // Endpoint configuration
 type Endpoint struct {
-	Host string `yaml:"host"`
-	Port int    `yaml:"port"`
-	Type string `yaml:"type"`
+	Host string        `yaml:"host"`
+	Port int           `yaml:"port"`
+	Type string        `yaml:"type"`
+	TTL  time.Duration `yaml:"ttl"`
```

Comment:
I'm not sure endpoint is the right place for the TTL. In the Registry's API, the TTL and endpoint are both attributes of the registered service instance.

Perhaps we can have the TTL (and later on, hearbeat frequency, maybe) under the `register` attribute:
```yaml
register:
	ttl: 60s
	heartbeat: 20s
```

And ideally, still support the boolean `register` flag that uses default values for ttl/heartbeat:
```yaml
register: true
```